src/storage/src/source/postgres/replication.rs

```diff
@@ -637,7 +637,11 @@ async fn raw_stream<'a>(
 
     // We use this timer to make sure we're always sending a standby update, regardless
     // of whether or not the upstream sends us periodic status updates.
-    let mut feedback_timer = tokio::time::interval(feedback_interval);
+    //
+    // Set a minimum interval to guard against feedback_interval being 0.
+    let minimum_interval = tokio::time::Duration::from_secs(1);
```

Comment:
How can this happen? If it can indeed happen then the code below is problematic since it will be sending a keepalive back for every single message received

Reply:
(I mean the original code, not the one you added)

Reply:
The value zero disables the timeout according to the docs https://www.postgresql.org/docs/current/runtime-config-replication.html#GUC-WAL-SENDER-TIMEOUT and in that case our code will do into the degenerate state of sending a ton of feedback messages back.